### PR TITLE
Wording tune up to TXT version of the registration email

### PR DESCRIPTION
**Pull request by yarikoptic**
In the attempt to establish a "human in the middle" approval of the accounts.

TODO:

- [ ] html version after TXT version agreed to be ok

### CODE REVIEW

File: dandiapi/api/mail.py
Comment (waxlamp) [inline dandiapi/api/mail.py:109]:
```suggestion
Once your account is activated you will be able to create Dandisets and upload data.
```

File: dandiapi/api/mail.py
Comment (waxlamp) [inline dandiapi/api/mail.py:106]: I used to put a double space after ending sentences but a few years ago I learned that it's no longer the prevailing practice.

```suggestion
It may take up to 24 hours for your account to be approved and activated. During the approval
```

**Review by waxlamp — CHANGES_REQUESTED** [review]

File: dandiapi/api/mail.py
Comment (yarikoptic) [reply dandiapi/api/mail.py:106]: "not prevailing" doesn't mean "not good" ;) Is it recommended for some specific reasons(s) to not do that?
Note: this is a text version of the email and IMHO double spacing helps to visually parse text.  HTML version will not have that

Comment (waxlamp) [reply dandiapi/api/mail.py:106]: The problem is, because it's "not prevailing", most rendering engines won't treat them the way typewriters did (i.e., present an extra space to make visual processing faster). I've seen things like a the double space getting "broken" across the ends of lines, etc.

For stuff like this, I like to just go with the prevailing notion of correctness (sort of like using Python Black to format source files) but honestly I don't have very strong feelings about this particular instance. If you want to keep it, that's ok with me--just resolve this thread so it's out of sight/out of mind 😸

**Comment by waxlamp** [comment]
@yarikoptic, could you finish this up and merge it so that #342 can be merged without causing a conflict here?

**Comment by yarikoptic** [comment]
> @yarikoptic, could you finish this up and merge it so that #342 can be merged without causing a conflict here?

well - we should finish/merge/deploy this only if we switch default behavior to have disabled users upon creation.  Is that doable/agreed upon to proceed?

**Comment by waxlamp** [comment]
> > @yarikoptic, could you finish this up and merge it so that #342 can be merged without causing a conflict here?
> 
> well - we should finish/merge/deploy this only if we switch default behavior to have disabled users upon creation. Is that doable/agreed upon to proceed?

In that case, let's hold off on this PR until after
1. @brianhelba merges his email template infrastructure changes, and
2. we can get more clarity on exactly how user registration should behave (including any other PRs necessary to implement what is reflected in this PR's explanatory changes).

If you are ok with that, could you change this PR's status to draft? After Brian's template changes land, it should be relatively easy to reconstitute your wording in the updated infrastructure.

**Comment by yarikoptic** [comment]
it is draft now (should have been from the beginning, thanks @waxlamp )

File: dandiapi/api/mail.py
Comment (waxlamp) [reply dandiapi/api/mail.py:106]: ping @yarikoptic

**Comment by waxlamp** [comment]
This has been solved via the user questionnaire.

**Pull request closed without merging**
